src/pages/Compare/index.tsx

```diff
@@ -131,14 +130,11 @@ export class CompareBase extends React.Component<Props> {
     const { addonId, compareInfo, version } = this.props;
 
     return (
-      <ContentShell
-        mainSidePanel={
-          version ? (
-            <FileTree onSelect={this.viewVersionFile} versionId={version.id} />
-          ) : (
-            this.renderLoadingMessageOrError(gettext('Loading file tree...'))
-          )
-        }
+      <VersionFileViewer
+        file={null}
+        onSelectFile={this.viewVersionFile}
+        showFileInfo={false}
```

Comment:
Does it make sense that we don't show file info on the compare page? That doesn't seem right to me. Do we have an issue open to address this?

Reply:
Good idea. I found an issue https://github.com/mozilla/addons-code-manager/issues/647